packages/admin/src/Resources/Pages/EditRecord.php

```diff
@@ -165,7 +165,7 @@ protected function getViewAction(): Action
 
     protected function getDeleteAction(): Action
     {
-        return DeleteAction::make('delete')
+        return DeleteAction::make()
             ->action(fn () => $this->delete());
```

Comment:
@danharrin I think it would be good if the action is also contained inside the action. That would clean up the pages even more, and make it easy to overwrite the action globally.

Reply:
Do you mean the `->action()` call? its unclear in GitHub which line you're referring to

Reply:
Yes the `->action()`. I guess this it because of compatibility with the tables package as we discussed on Discord?

Reply:
It's because people might already be overriding the `delete()` method or using hooks so we need to preserve that behaviour.

Reply:
I guess it would be better to use the `::configureUsing()` method and checking whether the class was extended with a `deleted()` method. See my original PR that was merged into this one.

Reply:
Shouldn't that be covered the way I did it? It looks a bit hackier, but would support old syntax only if overwritten, and use new default if not overwritten:

https://github.com/pxlrbt/filament/blob/f99f59bde307dbbb6670e3c5fc800a126a3815a7/packages/admin/src/Pages/Actions/DeleteAction.php#L65-L76

Reply:
The getDeleteAction() method needs to stay too as well, so I think this is cleaner.

Reply:
Alright.